corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -186,7 +186,10 @@ def get_all_case_properties(app):
 
 def get_usercase_properties(app):
     if is_usercase_in_use(app.domain):
-        return get_case_properties(app, [USERCASE_TYPE])
+        default_properties = {'first_name', 'last_name'}
```

Comment:
I think this should also contain `phone_number` and `username`, and eventually when we get it fixed, `name`. See [Accessing Phone Number and the User Name](https://confluence.dimagi.com/display/commcarepublic/Custom+User+Data#CustomUserData-AccessingPhoneNumberandtheUserName) and [User Case - Other Notes](https://confluence.dimagi.com/display/commcarepublic/User+Case#UserCase-OtherNotes). Maybe there are others as well? I couldn't find them.